### DISCUSSION TITLE
Trouble pushing to github

### DISCUSSION THREAD

**Discussion by emmats** [atsa-es/fish550-2023#14] [General]
I'm not sure why I can't commit and push my R script to my team's github folder. I generated my PAT and I checked - it is in place and correct. But when I commit my script and then Push I get the following error: 
>>> /usr/bin/git push origin HEAD:refs/heads/main
remote: Permission to atsa-es/fish550-2023.git denied to emmats.
fatal: unable to access 'https://github.com/atsa-es/fish550-2023/': The requested URL returned error: 403

**Comment by eeholmes** [comment]
@emmats Try again. You weren't on the ATSA-2023 team so didn't have push permission for the repo.

**Comment by emmats** [comment]
That fixed it! Thank you
